java/src/jmri/jmrix/can/cbus/swing/modeswitcher/SprogCbusModeSwitcherPane.java

```diff
@@ -0,0 +1,236 @@
+package jmri.jmrix.can.cbus.swing.modeswitcher;
+
+import java.awt.BorderLayout;
+import java.awt.event.ActionListener;
+import javax.swing.BorderFactory;
+import javax.swing.BoxLayout;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import jmri.GlobalProgrammerManager;
+import jmri.InstanceManager;
+import jmri.jmrix.can.CanListener;
+import jmri.jmrix.can.CanMessage;
+import jmri.jmrix.can.CanReply;
+import jmri.jmrix.can.CanSystemConnectionMemo;
+import jmri.jmrix.can.TrafficController;
+import jmri.jmrix.can.cbus.CbusDccProgrammerManager;
+import jmri.jmrix.can.cbus.CbusPreferences;
+import jmri.jmrix.can.cbus.CbusSend;
+import jmri.jmrix.can.cbus.node.CbusNode;
+import jmri.jmrix.can.cbus.node.CbusNodeTableDataModel;
+import jmri.util.JmriJFrame;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Mode Switcher to switch modes between programmer and command station.
+ *
+ * No matter what I tried, I cannot get the label text to wrap when added to a
+ * CanNamedPane, so this is implemented as a JmriJFrame.
```

Comment:
What kind of errors lead to this comment?  It looks strange having a Pane on the class name, but inheriting from a frame.  We either need to fix the class or change the name.

Reply:
Ah, that's left over from trying to use a CanNamedPane. See https://jmri-developers.groups.io/g/jmri/message/3326 I switched to a JmriJFrame instead . I have now renamed the class.